projects/components/src/summary-value/summary-value.component.ts

```diff
@@ -7,7 +7,7 @@ import { IconSize } from '../icon/icon-size';
   styleUrls: ['./summary-value.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="ht-summary-value" *ngIf="this.value" [htTooltip]="this.tooltipText">
+    <div class="ht-summary-value" *ngIf="!(this.value | htIsNil)" [htTooltip]="this.tooltipText">
```

Comment:
Why exactly do we need this pipe here?

Reply:
We wanna show this section if the string is empty, is it?

Reply:
Currently if value=0, it won't render. So the `isNil` checks for `value === undefined || value === null`